bin/run_demo.sh

```diff
@@ -48,6 +48,11 @@ if [ -z "$(butler query-datasets DATA_REPO/ raw | grep HSC)" ]; then
     butler define-visits DATA_REPO HSC --collections HSC/raw/all
 fi
 
+# Explicitly define a dataset type that uses the old style metadata definition.
+if [ -z "$(butler query-dataset-types DATA_REPO/ calibrate_metadata | grep -v results)"]; then
+    butler register-dataset-type DATA_REPO calibrate_metadata PropertySet band instrument detector physical_filter visit
```

Comment:
Should we include a note about removing this line and the associated test after some point? Do we have a deprecation/removal period for the old PropertySet repos?

Reply:
I think we can leave it in until something breaks because TaskMetadata is explicitly needed for something and PropertySet existing breaks it. Otherwise it's doing a nice job of adding another test for the storage class conversion codepath.